betty/cropper/templates/image.js

```diff
@@ -34,7 +34,10 @@
   }
 
   w.picturefill = function(elements, forceRerender) {
-
+    // It is sometimes desirable to scroll without loading images as we go.
+    if (w.pauseBettyCroppyPicturefill) {
+      return;
```

Comment:
That sounds good. :+1:

On Tue, Sep 15, 2015 at 11:08 AM, Andrew Kos <notifications@github.com>
wrote:

> In betty/cropper/templates/image.js
> <https://github.com/theonion/betty-cropper/pull/47#discussion_r39529826>:
>
> > @@ -34,7 +34,10 @@
> >    }
> >
> >    w.picturefill = function(elements, forceRerender) {
> > -
> > +    // It is sometimes desirable to scroll without loading images as we go.
> > +    if (w.pauseBettyCroppyPicturefill) {
> > +      return;
>
> Might be confusing since picturefill is a function that's called and not
> a persistent object, but can this go under a namespace, so it's use would
> be window.picturefill.paused() where paused is a getter/setter. This way
> we avoid further polluting the already extremely polluted window.
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/theonion/betty-cropper/pull/47/files#r39529826>.
>